dependencies.gradle

```diff
@@ -35,7 +35,6 @@ final def mod_dependencies = [
         'shadowfacts-forgelin-248453:2785465'                 : [project.debug_alchemistry, project.debug_industrial_foregoing],
         'ae2-extended-life-570458:4553942'                    : [project.debug_applied_energistics_2, project.debug_lazy_ae2],
         'arcane-archives-311357:3057332'                      : [project.debug_arcane_archives],
-        'guidebook-253874:2989594'                            : [project.debug_arcane_archives],
```

Comment:
you need guidebook as a dependency - its just the guidebook from curseforge doesnt satisfy it, as you need `gbook_snapshot`. i think this line should not be removed.

Reply:
Can we make this a `runtimeOnly` dependency? Because right now it's both which is pretty annoying.

Reply:
Although I will say that haven't ran into any compile-time failures (should have tested on a clean env though).